donate/payments/tasks.py

```diff
@@ -85,6 +85,7 @@ def process_subscription_charged_successfully(self, notification):
             'payment_frequency': constants.FREQUENCY_MONTHLY,
             'payment_method': constants.METHOD_PAYPAL if is_paypal else constants.METHOD_CARD,
             'transaction_id': last_tx.id,
+            'subscription_id': notification.subscription.id,
```

Comment:
This won't have any effect on its own - `send_transaction_to_basket` looks for specific keys in the dictionary that is passed to it, and `subscription_id` isn't one of them - we need to modify that function as well to pass through this parameter.

Reply:
Ah, thanks!